print/print-apps/oereb/mainpage.jrxml

```diff
@@ -219,7 +219,7 @@
 				<textElement verticalAlignment="Middle" markup="html">
 					<font fontName="Cadastra" size="9"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$P{RealEstate_LandRegistryArea} + " m<sup>2</sup>"]]></textFieldExpression>
+				<textFieldExpression><![CDATA[Integer.toString(Math.round(Float.parseFloat($P{RealEstate_LandRegistryArea}))) + " m<sup>2</sup>"]]></textFieldExpression>
```

Comment:
Why not:
```java
NumberFormat.getIntegerInstance().format(Float.parseFloat($P{RealEstate_LandRegistryArea}))
```

Reply:
Even better:
```
java.text.MessageFormat.format("{0,number,0} m<sup>2</sup>", $P{RealEstate_LandRegistryArea})
```
and RealEstate_LandRegistryArea should be a number ...

Reply:
👍